app/views/sign_up/select_email/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<% self.title = t('titles.select_email') %>
+<%= render AlertIconComponent.new(icon_name: :info_question, class: 'display-block margin-bottom-4', width: 54, height: 54) %>
+<h1><%= t('titles.select_email') %></h1>
+
+<p class="margin-top-4">
+  <%= I18n.t('help_text.select_preferred_email', sp: @sp_name, app_name: APP_NAME) %>
+</p>
+
+<%= simple_form_for(@select_email_form, url: sign_up_select_email_path) do |f| %>
+  <% @user_emails.each do |email, index| %>
+    <fieldset class="usa-fieldset">
+      <div class="usa-radio">
+        <%= radio_button_tag(
+              'select_email_form[selection]',
+              email,
+              email == @last_sign_in_email_address,
+              class: 'usa-radio__input usa-radio__input--bordered',
+            ) %>
+            <%= label_tag(
+                  "select_email_form_selection_#{email}",
```

Comment:
Is `email` here the literal email address? I'm not sure how well I'd expect that to work as an HTML ID.

For example, there's this guidance from MDN:

>However, to avoid inadvertent errors, only [ASCII](https://developer.mozilla.org/en-US/docs/Glossary/ASCII) letters, digits, '_', and '-' should be used

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

Reply:
it gets cleaned up by Rails so it's recast as `select_email_form_selection_user_email.com` I don't know if the period is maybe problematic though.